habitat-lab/habitat/tasks/rearrange/actions/humanoid_actions.py

```diff
@@ -193,3 +190,130 @@ def reset(self, *args, **kwargs):
             self._prev_ep_id = self._task._episode_id
             self.skill_done = False
         self.hand_pose_iter = 0
+
+
+@registry.register_task_action
+class HumanoidPickObjIdAction(HumanoidPickAction):
+    def __init__(self, *args, task, **kwargs):
+        config = kwargs["config"]
+        HumanoidJointAction.__init__(self, *args, **kwargs)
+        self.vdb = None
+
+        self.humanoid_controller = self.lazy_inst_humanoid_controller(
+            task, config
+        )
+
+        self._task = task
+        self._prev_ep_id = None
+        self._targets = {}
```

Comment:
Could you add a type annotation or change the variable name to describe what the set is supposed to contain?

Reply:
Unused variable, removed